packages/fxa-auth-client/lib/crypto.ts

```diff
@@ -26,8 +71,9 @@ export async function getCredentials(email: string, password: string) {
   const quickStretchedRaw = await crypto.subtle.deriveBits(
     {
       name: 'PBKDF2',
-      salt: encoder().encode(`${NAMESPACE}quickStretch:${email}`),
-      iterations: 1000,
+      salt: encoder().encode(salt),
+      // TBD: Iterations
+      iterations: /quickStretchV2:/.test(salt) ? 100000 : 1000,
```

Comment:
`parseSalt` returns a value with a version number.  Maybe use that instead of a regex?

The number of iterations being bound to the salt version doesn't feel right to me.  But maybe it's just a matter of naming.  I suppose that we'll always update the two params, salt and number of iteration, together.

Let's use 600000+ iterations?